modules/serving-transport-encryption-choice-of-a-certificate-issuer.adoc

```diff
@@ -0,0 +1,49 @@
+:_mod-docs-content-type: CONCEPT
+[id="serving-transport-encryption-choice-of-a-certificate-issuer_{context}"]
+= Choice of a certificate issuer
+
+[NOTE]
+====
+Issuers refer to cert-manager `Issuers` and `ClusterIssuers`. They represent certificate authorities (CAs) that can generate signed certificates by honoring certificate signing requests.
+====
```

Comment:
```suggestion
Issuers refer to cert-manager issuers and cluster issuers. They represent certificate authorities (CAs) that can generate signed certificates by honoring certificate signing requests.
```
I don't think that this really needs to be a note. Also, formatting of these components should be consistent with other product docs:
- "issuers", per https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/security_and_compliance/cert-manager-operator-for-red-hat-openshift#cert-manager-issuer-types_cert-manager-operator-about
- "cluster issuers" per https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/security_and_compliance/cert-manager-operator-for-red-hat-openshift#cert-manager-acme-http01_cert-manager-operator-issuer-acme